src/ui/components/SecondaryToolbox/react-devtools/components/HooksRenderer.tsx

```diff
@@ -33,7 +33,9 @@ export function HooksRenderer({
     return null;
   }
 
-  return object.preview.properties.map((property, index) => (
+  const objectProperties = object.preview.properties.filter(property => property.object);
```

Comment:
I feel like a utility function that filters out only Array elements from an `ObjectPreview` might be better here.

Reply:
This change needs more description/context via an inline comment.

Reply:
TBH I don't like this approach we're using (neither here nor the fix yesterday). 

I think we should instead be checking to see if the property is enumerable (via `.flags`)